src/components/Portal.tsx

```diff
@@ -72,3 +72,23 @@ const DefaultPortal = createPortalGroup()
 export const Provider = DefaultPortal.Provider
 export const Outlet = DefaultPortal.Outlet
 export const Portal = DefaultPortal.Portal
+
+// when in native sheets, we want the default Portal to move to inside the sheet for children of the sheet
+export const DefaultPortalContext = React.createContext(DefaultPortal.Portal)
```

Comment:
Is "default" the right word here? I feel like we could just call it `PortalContext`. The default is `DefaultPortal`, which is aptly named in this file. But this context overrides, and in the spirit of "not thinking about it", I think `usePortal` or like `useParentPortal` might make more sense? No strong feelings, just spitballing.

Reply:
useDefaultPortal makes sense in the context of it maybe being passed in via prop instead. you shouldn't need to know that the default portal might get overridden if in a sheet